### PR TITLE
fixing code styling for pinot-core

### PR DESCRIPTION
**Pull request by xiangfu0**
## Description
fixing code styling for pinot-core
## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

File: pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunction.java [hunk 1]
```diff
@@ -255,6 +255,7 @@ public Dictionary getDictionary() {
         case BYTES:
           byte[][] bytesValues = transformToBytesValuesSV(projectionBlock);
           ArrayCopyUtils.copy(bytesValues, _stringValuesSV, length);
+          break;
```
Comment (xiangfu0) [inline pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunction.java:258]: bug found

File: pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BinaryOperatorTransformFunction.java [hunk 2]
```diff
@@ -115,7 +116,8 @@ private void fillResultArray(ProjectionBlock projectionBlock) {
             break;
```
Comment (xiangfu0) [inline pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BinaryOperatorTransformFunction.java:116]: bug found

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountBitmapMVAggregationFunction.java [hunk 3]
```diff
@@ -86,6 +86,7 @@ public void aggregate(int length, AggregationResultHolder aggregationResultHolde
             valueBitmap.add(Float.hashCode(value));
           }
         }
+        break;
```
Comment (xiangfu0) [inline pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountBitmapMVAggregationFunction.java:89]: bug found.

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountMVAggregationFunction.java [hunk 4]
```diff
@@ -93,6 +93,7 @@ public void aggregate(int length, AggregationResultHolder aggregationResultHolde
             floatSet.add(value);
           }
         }
+        break;
```
Comment (xiangfu0) [inline pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountMVAggregationFunction.java:96]: bug found.

File: pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java [hunk 5]
```diff
@@ -134,7 +134,8 @@ void reloadAllSegments(String tableNameWithType)
   ZkHelixPropertyStore<ZNRecord> getPropertyStore();
 
   /**
-   * Returns the segment uploader, which uploads a llc segment to the destination place and returns the url of uploaded segment file. Servers utilize segment uploader to upload llc segment to segment store.
+   * Returns the segment uploader, which uploads a llc segment to the destination place and returns the url of uploaded segment file.
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java:137]: This seems still over 120 characters. Shall we try to make all of them under 120 characters if possible? Same for other files

Comment (xiangfu0) [reply pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java:137]: We have a checkstyle hard failure limit of 160. 
For now, I break the line at 140, tried with 120, and it makes many expressions look a bit ugly with intentions.

Comment (Jackie-Jiang) [reply pinot-core/src/main/java/org/apache/pinot/core/data/manager/InstanceDataManager.java]: Suggest keeping the 120 because everyone has been following the code format

File: pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunction.java [hunk 6]
```diff
@@ -41,6 +41,7 @@
 
 
 /**
+ * CHECKSTYLE:OFF
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunction.java:44]: Why is this needed?

File: pinot-core/src/main/java/org/apache/pinot/core/util/SortedRangeIntersection.java [hunk 7]
```diff
@@ -60,27 +62,28 @@
         }
       }
       // move all pointers forward such that range they point to contain maxHead
-      for (int i = 0; i < sortedRangeSetList.size(); i++) {
-        if (i == maxHeadIndex) {
+      int j = -1;
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/util/SortedRangeIntersection.java:65]: Why do we need to change this?

File: pinot-core/src/test/java/org/apache/pinot/core/data/table/TableResizerTest.java [hunk 8]
```diff
@@ -65,11 +65,47 @@
   public void setUp() {
     //@formatter:off
     _records = Arrays.asList(
-        new Record(new Object[]{"a", 10, 1.0, 10.0, 100.0, new IntOpenHashSet(new int[]{1}), new AvgPair(10, 2) /* 5 */}),
-        new Record(new Object[]{"b", 10, 2.0, 20.0, 200.0, new IntOpenHashSet(new int[]{1, 2}), new AvgPair(10, 3) /* 3.33 */}),
-        new Record(new Object[]{"c", 200, 3.0, 30.0, 300.0, new IntOpenHashSet(new int[]{1, 2}), new AvgPair(20, 4) /* 5 */}),
-        new Record(new Object[]{"c", 50, 4.0, 30.0, 200.0, new IntOpenHashSet(new int[]{1, 2, 3}), new AvgPair(30, 10) /* 3 */}),
-        new Record(new Object[]{"c", 300, 5.0, 20.0, 100.0, new IntOpenHashSet(new int[]{1, 2, 3, 4}), new AvgPair(10, 5) /* 2 */})
+        new Record(new Object[]{
+            "a",
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/data/table/TableResizerTest.java:69]: Don't need to fully expand it

File: pinot-core/src/test/java/org/apache/pinot/core/data/table/TableResizerTest.java [hunk 9]
```diff
@@ -65,11 +65,47 @@
   public void setUp() {
     //@formatter:off
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/data/table/TableResizerTest.java:66]: Can we remove this annotation?

File: pinot-core/src/test/java/org/apache/pinot/core/operator/filter/predicate/RangeOfflineDictionaryPredicateEvaluatorTest.java [hunk 10]
```diff
@@ -35,7 +35,8 @@
 
   @Test
   public void testRanges() {
-    int rangeStart, rangeEnd;
+    int rangeStart;
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/operator/filter/predicate/RangeOfflineDictionaryPredicateEvaluatorTest.java:38]: (nit) Move them into each block?

File: pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java [hunk 11]
```diff
@@ -29,6 +29,7 @@
 
 
 public class CaseTransformFunctionTest extends BaseTransformFunctionTest {
+
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java:32]: (nit) remove

File: pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java [hunk 12]
```diff
@@ -45,29 +45,35 @@
     List<Object[]> inputs = new ArrayList<>();
 
     // max in array (returns SV INT)
+    //CHECKSTYLE:OFF
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java:48]: Why do we need to turn it off?

File: pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java [hunk 13]
```diff
@@ -45,29 +45,35 @@
...
     groovyTransformFunction = String
         .format("groovy('{\"returnType\":\"INT\", \"isSingleValue\":true}', " + "'arg0.toList().max()', " + "%s)",
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java:50]: Remove the string concat? Same for other places

File: pinot-core/src/test/java/org/apache/pinot/core/query/pruner/SelectionQuerySegmentPrunerTest.java [hunk 14]
```diff
@@ -99,8 +99,7 @@ public void testSelectionOnly() {
 
   @Test
   public void testSelectionOrderBy() {
-    List<IndexSegment> indexSegments = Arrays.asList( //
-        getIndexSegment(0L, 10L, 10),     // 0
+    List<IndexSegment> indexSegments = Arrays.asList(getIndexSegment(0L, 10L, 10),     // 0
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/core/query/pruner/SelectionQuerySegmentPrunerTest.java:102]: Move this to next line?

File: pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java [hunk 15]
```diff
@@ -582,7 +582,18 @@ private void testDistinctInnerSegmentHelper(String[] queries, boolean isPql) {
   public void testDistinctInnerSegment() {
     //@formatter:off
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java:583]: Can we remove this?

File: pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java [hunk 16]
```diff
@@ -582,7 +582,18 @@ private void testDistinctInnerSegmentHelper(String[] queries, boolean isPql) {
...
     testDistinctInnerSegmentHelper(
-        new String[]{"SELECT DISTINCT(intColumn, longColumn, floatColumn, doubleColumn, stringColumn, bytesColumn) FROM testTable LIMIT 10000", "SELECT DISTINCT(stringColumn, bytesColumn, floatColumn) FROM testTable WHERE intColumn >= 60 LIMIT 10000", "SELECT DISTINCT(intColumn, rawBytesColumn) FROM testTable ORDER BY rawBytesColumn LIMIT 5", "SELECT DISTINCT(ADD(intColumn, floatColumn), stringColumn) FROM testTable WHERE longColumn < 60 ORDER BY stringColumn DESC, ADD(intColumn, floatColumn) ASC LIMIT 10", "SELECT DISTINCT(floatColumn, longColumn) FROM testTable WHERE stringColumn = 'a' ORDER BY longColumn LIMIT 10"},
+        new String[] {
+            //CHECKSTYLE:OFF
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java:586]: Do we need it?

File: pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java [hunk 17]
```diff
@@ -601,9 +612,23 @@ public void testDistinctInnerSegment() {
   @Test
   public void testNonAggGroupByRewriteToDistinctInnerSegment() {
     //@formatter:off
+    //CHECKSTYLE:OFF
```
Comment (Jackie-Jiang) [inline pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java:615]: Same

**Review by Jackie-Jiang — APPROVED** [review]

File: pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunction.java (hunk 6 above)
Comment (xiangfu0) [reply pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunction.java:44]: there is `arg0` in the comments which is regex'ed from checkstyle as warning

File: pinot-core/src/main/java/org/apache/pinot/core/util/SortedRangeIntersection.java (hunk 7 above)
Comment (xiangfu0) [reply pinot-core/src/main/java/org/apache/pinot/core/util/SortedRangeIntersection.java:65]: checkstyle doesn't allow modify the for loop value inside the loop block

File: pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java (hunk 12 above)
Comment (xiangfu0) [reply pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/GroovyTransformFunctionTest.java:48]: same issue of `arg0`, `arg1`...
